<?xml version="1.0"?>

<launch>

    <!--Launch Gazebo, initialize the environment, Spawn the Quadrotor, and the quadrotor controllers-->
    <include file="$(find ORTINa)/launch/spawn_quadrotor.launch"/>

    <!-- Run the map server -->
    <include file="$(find ORTINa)/launch/map_server_init.launch"/>
   
    <!-- Path smoothing-->
    <include file="$(find ORTINa)/launch/path_smoothing.launch"/>

    <!-- Kalman Filter --> 
    <node pkg="ORTINa" name="kalmanFilter" type="kalmanFilter.py" output="screen"/>

</launch>
